nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/kafka-config.xml.nxftl

```diff
@@ -8,20 +8,22 @@
       <producer>
         <property name="bootstrap.servers">${kafka.bootstrap.servers}</property>
         <property name="default.replication.factor">${kafka.default.replication.factor}</property>
-<#if "${kafka.ssl}" == "true">
-<#if "${kafka.sasl.enabled}" == "true">
+<#if ((kafka.sasl.enabled)!"false") == "true" || ((kafka.ssl)!"false") == "true">
         <property name="security.protocol">${kafka.security.protocol}</property>
+</#if>
+<#if "${kafka.sasl.enabled}" == "true">
         <property name="sasl.mechanism">${kafka.sasl.mechanism}</property>
         <property name="sasl.jaas.config">${kafka.sasl.jaas.config}</property>
-<#else>
-        <property name="security.protocol">SSL</property>
```

Comment:
It seems like with your change we won't have anymore `security.protocol` set to `SSL` when `kafka.ssl` is `true`.
And the `kafka.security.protocol` property never seems to be set to `SSL` either in the nuxeo code.

Reply:
> It seems like with your change we won't have anymore security.protocol set to SSL when kafka.ssl is true.

This does set `security.protocol` when `kafka.ssl` is true, (or did you mean the opposite ?):
```
<#if "${kafka.sasl.enabled}" == "true" || "${kafka.ssl}" == "true">
        <property name="security.protocol">${kafka.security.protocol}</property>
```

`kafka.security.protocol` is set in `/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults` so shouldn't this be taken into account if the line is not present in the user's `nuxeo.conf`